plugins/artnet/Makefile.mk

```diff
@@ -11,9 +11,15 @@ plugins_artnet_libolaartnetnode_la_SOURCES = \
     plugins/artnet/ArtNetNode.h
 plugins_artnet_libolaartnetnode_la_LIBADD = common/libolacommon.la
 
+# Plugin description is generated from README.md
+BUILT_SOURCES += plugins/artnet/ArtNetPluginDescription.h
```

Comment:
I tried that at the beginning, but it didn't work because it was being built too late, I guess. After switching to `BUILT_SOURCES` in bc8667a, that part worked at least.